lib/printer.rb

```diff
@@ -1,6 +1,9 @@
+# require './color'
+require 'colorize'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.